app/src/main/res/values-v21/strings.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources></resources>
```

Comment:
I think this file is of no use. If you are keeping it than add EOF.

Reply:
Removed